src/main/java/org/kainos/ea/cli/Role.java

```diff
@@ -0,0 +1,26 @@
+package org.kainos.ea.cli;
+    public enum Role{
```

Comment:
Would this be better as en enum or being part of the database? With the current implementation is there any way the IDs in the database could get out of sync with the values in the `enum`?